from_cpython/Modules/_ctypes/_ctypes.c

```diff
@@ -2044,13 +2044,20 @@ PyCSimpleType_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
             break;
         }
 
-        if (ml) {
+        // TODO: Pyston change:
+        // For now, don't run this code path because we don't support descriptors.
```

Comment:
fyi we support descriptors in general, it looks like we just don't support a couple of the capi functions for creating some special types of them.  though we do support those types (members, getsets, and classmethods).